schema/codelists/addressType.csv

```diff
@@ -1,7 +1,8 @@
-code,title,description,technical note
-placeOfBirth,Place of birth
-home,Home address
-residence,Residential address
-registered,Registered address
-service,Service address
-alternative,Alternative address
\ No newline at end of file
+code,title,description,technical note,
+placeOfBirth,Place of birth,The place where a person was born. This must only be used to describe natural persons. Only a town and country are required.,,
+home,Home address,The address of the house or apartment which is someone's main place of residence. This must only be used for natural persons.,,
+residence,Residential address,An address of a house or apartment where someone lives but that is not their home address. This must only be used for natural persons.,,
+registered,Registered address,An official address for delivering statutory mail and legal notices which must be provided to company registers in the UK and other jurisdictions. This must only be used to describe entities.,,
```

Comment:
An official address which must be provided to the company register <s>in the UK and other jurisdictions</s>for delivering statutory mail and legal notices. This must only be used to describe entities.

Reply:
Edited as per suggestion above. Should this also be changed from 'must' to 'may'?